components/support/test/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
 
     lintOptions {
         lintConfig file("lint.xml")
+        disable 'InvalidPackage'
```

Comment:
I had to suppress this check to avoid  the lint error.
To be able to reuse Robolectric code.
![image](https://user-images.githubusercontent.com/773158/46838958-9ba07d00-cd89-11e8-850b-51528c344a28.png)

Reply:
See lint.xml in the module. You can just list those jars to be excluded. There should already be one listed in the file.

Reply:
👍

Reply:
Done! 🕺